inception-workload-dynamic/src/main/java/de/tudarmstadt/ukp/inception/workload/dynamic/annotation/DynamicWorkflowDocumentNavigationActionBarExtension.java

```diff
@@ -73,16 +95,18 @@ public int getPriority()
     }
 
     @Override
-    public boolean accepts (AnnotationPageBase aPage)
+    public boolean accepts(AnnotationPageBase aPage)
     {
         // #Issue 1813 fix
         if (aPage.getModelObject().getProject() == null) {
             return false;
         }
-
-        return DYNAMIC_WORKLOAD_MANAGER_EXTENSION_ID.equals(workloadManagementService.
-            getOrCreateWorkloadManagerConfiguration(aPage.getModelObject().getProject())
-            .getType());
+        // Curator are excluded from the feature
+        return DYNAMIC_WORKLOAD_MANAGER_EXTENSION_ID
+                .equals(workloadManagementService.loadOrCreateWorkloadManagerConfiguration(
```

Comment:
You can fetch the `aPage.getModelObject()` once, store it in a local variable and then use it. Makes sense if you use a value more than once.